Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml

```diff
@@ -1235,322 +1253,377 @@ tasks:
+    view: |-
+      {
+        "position": {
+          "x": 1350,
+          "y": 1720
+        }
+      }
+  "89":
+    continueonerrortype: ""
+    id: "89"
     ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-    isoversize: false
     isautoswitchedtoquietmode: false
-  '89':
-    id: '89'
-    taskid: 49c685f0-c18c-4536-886b-b9c0f7a7daad
-    type: playbook
-    task:
-      id: 49c685f0-c18c-4536-886b-b9c0f7a7daad
-      version: -1
-      name: Cortex ASM - ServiceNow ITSM Enrichment
-      type: playbook
+    isoversize: false
+    loop:
+      exitCondition: ""
       iscommand: false
-      brand: ''
-      description: ''
-      playbookId: Cortex ASM - ServiceNow ITSM Enrichment
+      max: 0
+      wait: 1
     nexttasks:
       '#none#':
-      - '62'
+      - "62"
+    note: false
+    quietmode: 0
     scriptarguments:
       search_terms:
         complex:
-          root: alert.asmsystemids
           filters:
-          - - operator: isEqualString
-              left:
+          - - left:
+                iscontext: true
                 value:
                   simple: alert.asmsystemids.type
-                iscontext: true
+              operator: isEqualString
               right:
                 value:
                   simple: ASSET-NAME
-            - operator: isEqualString
-              left:
+            - left:
+                iscontext: true
                 value:
                   simple: alert.asmsystemids.type
-                iscontext: true
+              operator: isEqualString
               right:
                 value:
                   simple: ASSET-ID
+          root: alert.asmsystemids
           transformers:
-          - operator: getField
-            args:
+          - args:
               field:
                 value:
                   simple: id
+            operator: getField
           - operator: uniq
-          - operator: replaceMatch
-            args:
+          - args:
               regex:
                 value:
                   simple: '[nN]/[aA]'
               replaceWith: {}
-          - operator: RemoveEmpty
-            args:
+            operator: replaceMatch
+          - args:
               empty_values: {}
               remove_keys: {}
+            operator: RemoveEmpty
     separatecontext: true
-    continueonerrortype: ''
-    loop:
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 34eeacf7-2293-44bb-8ac3-557d042a5f41
       iscommand: false
-      exitCondition: ''
-      wait: 1
-      max: 0
+      name: Cortex ASM - ServiceNow ITSM Enrichment
+      playbookId: Cortex ASM - ServiceNow ITSM Enrichment
+      type: playbook
+      version: -1
+      description: ''
+    taskid: 34eeacf7-2293-44bb-8ac3-557d042a5f41
+    timertriggers: []
+    type: playbook
     view: |-
       {
         "position": {
           "x": 460,
           "y": 2605
         }
       }
-    note: false
-    timertriggers: []
+  "91":
```

Comment:
Please add a header for `Prisma Enrichment` and `Directory Enrichment` above the two task that perform this.

Reply:
Added the headers